### DISCUSSION TITLE
Homepage rewrite

### DISCUSSION THREAD

**Discussion by adamsoffer** [vercel/next.js#10763] [Help] [answered]
What's the recommended way to create a site "frontpage"? I'm using dynamic routes (`[slug].tsx`) and `getStaticPaths` and I'm trying to instruct next to serve the same static page when a user visits `/` or `/home`. I'd like to avoid creating two separate pages to achieve this.

**Comment by andreisena** [comment, marked as answer]
There is a experimental feature for this. You can check here #9081.
It allows you to create redirects like this:

```js
// next.config.js
module.exports = {
  async rewrites() {
    return [
      {
        source: "/home",
        destination: "/",
      }
    ];
  }
};
```
